pkg/virtual/apiexport/builder/build.go

```diff
@@ -185,15 +176,116 @@ func BuildVirtualWorkspace(
 
 			return apiReconciler, nil
 		},
-		Authorizer: getAuthorizer(kubeClusterClient),
+		Authorizer: newAuthorizer(kubeClusterClient),
 	}
 
 	return []rootapiserver.NamedVirtualWorkspace{
-		{Name: VirtualWorkspaceName, VirtualWorkspace: boundWorkspaceContent},
+		{Name: VirtualWorkspaceName, VirtualWorkspace: boundOrClaimedWorkspaceContent}, // this must come first because a claim will show all bindings, not only those for the export
+		{Name: apiBindingsName, VirtualWorkspace: apiBindings},
 	}, nil
 }
 
-func getAuthorizer(client kubernetes.ClusterInterface) authorizer.AuthorizerFunc {
+var resolver = requestinfo.NewFactory()
+
+func isAPIBindingRequest(path string) bool {
+	info, err := resolver.NewRequestInfo(&http.Request{URL: &url.URL{Path: path}})
+	if err != nil {
+		return false
+	}
+	return info.IsResourceRequest && info.APIGroup == apisv1alpha1.SchemeGroupVersion.Group && info.Resource == "apibindings"
+}
+
+func digestUrl(urlPath, rootPathPrefix string) (genericapirequest.Cluster, dynamiccontext.APIDomainKey, string, bool) {
+	if !strings.HasPrefix(urlPath, rootPathPrefix) {
+		return genericapirequest.Cluster{}, "", "", false
+	}
+
+	// Incoming requests to this virtual workspace will look like:
+	//  /services/apiexport/root:org:ws/<apiexport-name>/clusters/*/api/v1/configmaps
+	//                     └────────────────────────┐
+	// Where the withoutRootPathPrefix starts here: ┘
```

Comment:
love those comments 👍 they help a lot!

Reply:
thanks @stevekuznetsov. He had shown us the light.